lib/ansible/module_utils/docker/common.py

```diff
@@ -463,18 +465,15 @@ def _get_minimal_versions(self, option_minimal_versions, ignore_params=None):
                     else:
                         usg = 'set %s option' % (option, )
                     if not support_docker_api:
-                        msg = 'docker API version is %s. Minimum version required is %s to %s.'
+                        msg = 'Docker API version is %s. Minimum version required is %s to %s.'
                         msg = msg % (self.docker_api_version_str, data['docker_api_version'], usg)
                     elif not support_docker_py:
+                        msg = "Docker SDK for Python version is %s. Minimum version required is %s to %s."
                         if LooseVersion(data['docker_py_version']) < LooseVersion('2.0.0'):
-                            msg = ("docker-py version is %s. Minimum version required is %s to %s. "
-                                   "Consider switching to the 'docker' package if you do not require Python 2.6 support.")
+                            msg = (msg + " Hint: if you do not need Python 2.6 support, try `pip uninstall "
+                                   "docker-py` followed by `pip install docker`.")
                         elif self.docker_py_version < LooseVersion('2.0.0'):
-                            msg = ("docker-py version is %s. Minimum version required is %s to %s. "
-                                   "You have to switch to the Python 'docker' package. First uninstall 'docker-py' before "
-                                   "installing 'docker' to avoid a broken installation.")
-                        else:
-                            msg = "docker version is %s. Minimum version required is %s to %s."
+                            msg = (msg + " Try `pip uninstall docker-py` followed by `pip install docker`.")
```

Comment:
Sometimes you add "Hint:" and sometimes not. I think this should be consistent. Maybe this error message should be generated in dedicated method so it is consistent across all modues?

Reply:
I use `Hint:` only in the case that the user currently uses `docker-py` and the module also works with `docker-py` (albeit a newer version < 2.0); in case the module requires `docker` (>= 2.0) and the user has `docker-py` installed, or an older version of `docker`, it will not add `Hint:` but give precise instructions on how to upgrade.

But yeah, these messages could be grouped better. I'll add something for that.